task_01.rst

```diff
@@ -0,0 +1,84 @@
+
+  The School's online courses take full advantage of emerging education technolo  gies and interactive formats. The rich online learning environment offers stud  ents the opportunity to interact with instructors and their classmates in a nu  mber o  f ways including email, discussion boards, blogs, ePortfolios, and ins  tant messa  ging.
+
+Online Bachelor's Degrees
+-------------------------
+
+- Lists Of Degrees
+
+  + Communication and Media (B.A.)
+  + Disability Studies (B.A.)
+  + Information Systems (B.S.)
+  + Psychology (B.A.)
+  + Sociology (B.A.)
+  + Business (B.S.)
+  + Health Information Management (B.S.)
+  + Nursing (R.N. to B.S.)
+
+
+Online Master's Degrees
+-----------------------
+
+- Lists Of Degrees
+
+  + Business Management and Leadership (M.S.)
+  + Data Analytics (M.S.)
+  + Disability Services in Higher Education (M.S.)
+  + Disability Studies (M.A.)
+  + Psychology (M.A.)
+   
+
+Online Certificates
+-------------------
+
+- Lists Of Certificates
+
+  + Disability Studies (Graduate Certificate)
+  + Immigration Law (Graduate Certificate)
+  + Medical Coding (Undergraduate Certificate)
+  + Project Management (Graduate Certificate)
+
+Student Services
+-----------------
+
+Home>>Student Services
+  
+  The CUNY School of Professional Studies offers a comprehensive array of services to assist students to achieve their academic goals. From tutoring to opportunities to  study abroad, the Student Services Office is available to work with students to accommodate their needs and interests. 
+
++---------------------------+---------------------+
+|**Student Services**       |**Student Life**     | 
++===========================+=====================+
+|Academic Resources         |Around SPS and CUNY  |
++---------------------------+---------------------+
+|Career Services            |Health Services      |
++---------------------------+---------------------+
+|Disability Services        |Living in NYC        |    
++---------------------------+---------------------+
+|Help Desk                  |Register to Vote     |
++---------------------------+---------------------+
+|Library Services           |Student ID's         |    
++---------------------------+---------------------+ 
+|Military and Veterans      |Study Abroad         |      
+|Services                   |                     |
++---------------------------+---------------------+
+|Tutoring Services          |Virtual Campus       | 
++---------------------------+---------------------+
+|2014-2015 Student Technology Fee Plan            |
++-------------------------------------------------+
+
+
+Contact CUNY for Assistance 
+------------------------------------
+
+<<<If you need to contact the Office of Student Services, please send an email to studentservices@sps.cuny.edu.>>>
```

Comment:
Almost perfect until here. First off your underline is too long for the text. Secondly, what this is is a *note* eg:

```
.. note::

    If you need...
```

Otherwise, it's fantastic.